orchagent/aclorch.h

```diff
@@ -329,8 +334,10 @@ class AclOrch : public Orch, public Observer
 
     bool processAclTableType(string type, acl_table_type_t &table_type);
     bool processAclTableStage(string stage, acl_stage_type_t &acl_stage);
-    bool processPorts(string portsList, std::function<void (sai_object_id_t)> inserter);
+    bool processPorts(AclTable &aclTable, string portsList, std::function<void (sai_object_id_t)> inserter);
+    bool processPendingPort(AclTable &aclTable, string portAlias, std::function<void (sai_object_id_t)> inserter);
     bool validateAclTable(AclTable &aclTable);
+    sai_object_id_t getValidPortId(string alias, Port port);
```

Comment:
>getValidPortId [](start = 20, length = 14)

to reflect the true meaning, it is better to rename it to getBindPortId. Besides, it makes more sense to move this into portsyncd since it is retrieving a port property.

Reply:
renamed and move it to portOrch.